src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--wed calendar--month-length--29">
+      <div class="calendar__days"></div>
```

Comment:
This is one day, not a container for days

```suggestion
      <div class="calendar__day"></div>
```